tests/app/Listeners/PluginError/RecordPluginErrorInBugsnagTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Listeners\PluginError;
+
+use App\BaseUnitTestCase;
+use App\Events\PluginErrorReceivedEvent;
+use App\Models\PluginError\PluginError;
+
+class RecordPluginErrorInBugsnagTest extends BaseUnitTestCase
```

Comment:
I'm not sure this test class has much value?

Reply:
I'm a stickler for high test coverage, that's why it exists xD

Reply:
If my Bugsnag comment above is addressed it should make this test (and the event) redundant